AndroidTrainingModule/res/layout/relative_layout_activity.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+    
+    <TextView 
+        android:id="@+id/main_textView"
+        android:gravity="center"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="30sp"
+        android:text="@string/hello_world"/>
+
+    <EditText
+        android:id="@+id/main_editText"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
```

Comment:
Same with setting your textSize, you may also use dimens.xml in setting constant width and height.